docs/toc.json

```
@@ -197,6 +197,11 @@
         "slug": "/channels/telegram/",
         "title": "Telegram"
       }
```

```suggestion
      },
```

Ini jangan lupa ditambah koma ya, build-nya jadi error karena JSON-nya gak bener. 😅